data/RI/incentives.json

```diff
@@ -245,6 +245,32 @@
       "es": "Un reembolso adicional de $1,500 por un vehículo eléctrico usado para los calificadores de LIHEAP. Por orden de llegada."
     }
   },
+  {
+    "id": "RI-13",
+    "authority_type": "state",
+    "authority": "ri-oer",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "ebike",
```

Comment:
I added this to the API because of Tom's comment here: https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1708462830835669?thread_ts=1708461014.290839&cid=C04E4THEGMT

but the tests are failing with `launched states do not have any values that we don't support in the frontend > ebike not a launched technology
  in RI-13 and cannot be used in state: RI`. is this expected? should I update ebikes to be a launched item? https://github.com/rewiringamerica/api.rewiringamerica.org/blob/main/src/data/types/items.ts#L24

Reply:
Not sure about this one -- I don't have a sense of the full implications of making an item "launched" (I don't even remember seeing this code change go by). I think this is a question for Slack.

Reply:
followed up in [slack](https://rewiringameri-g3x1100.slack.com/archives/C04E4THEGMT/p1708539293768709?thread_ts=1708461014.290839&cid=C04E4THEGMT) with @datu925 . sounds like the beta vs launched distinction was to ensure that all launched items have a `more_info` url, but we want to move away from that (Dan is handling removing that requirement). we also want to ensure that all launched items have Spanish translations.

based on that this PR is blocked by:
1. Spanish translations for the 2 new ebike rebates ([ticket created for CCCE here](https://app.asana.com/0/1205381742684374/1206659011068825/f))
2. Dan's updates to enable us to "launch" ebikes without the `more_info` url

Reply:
https://github.com/rewiringamerica/api.rewiringamerica.org/pull/304 landing should unblock #2.